c/src/ml-api-inference-single.c

```diff
@@ -1088,6 +1088,20 @@ ml_single_open_custom (ml_single_h * single, ml_single_preset * info)
     g_object_set (filter_obj, "custom", info->custom_option, NULL);
   }
 
+  if (single_h->klass && info->invoke_async) {
+    if (info->invoke_async_cb != NULL && info->invoke_async_data!= NULL) {
+      NNSFilterInvokeAsyncCallback invoke_async_cb =
+          (NNSFilterInvokeAsyncCallback) info->invoke_async_cb;
+      single_h->klass->set_invoke_async_callback (single_h->filter,
```

Comment:
We need update of nnsteamer into tizen first. After update, it is hard to fix both nnstreame and api repo.
So, let's focus the function and callback of async-invoke in nnstreamer repo now.